shumai/tensor/tensor.ts

```diff
@@ -577,6 +574,11 @@ export class Tensor {
     return t
   }
 
+  untidy() {
+    _tidyTracker?.delete(this.ptr)
```

Comment:
Callers should not be accessing `_tidyTracker` directly. This helper makes more sense. Considered `unref` for naming but opted for more explicit/descriptive naming.